tracer/test/snapshots/Security.AspNetWebFormsAsmData.Classic.enableSecurity=False.__test=blocking-ips_url=_.verified.txt

```diff
@@ -0,0 +1,110 @@
+[
+  {
```

Comment:
I'm surprised that there are four separate traceId's, when I expected two. Is this expected? If not, we should followup on this later

Reply:
It seem to treat the requests for `Resource: GET /default.aspx` and `Resource: GET /` as two different requests. As they both end in 200, this probably isn't redirect but some quirk of ASP.NET web forms the tracer doesn't handle correctly? I don't think it's related to this PR.

Reply:
I move over to calling `/default.aspx` and get the correct number of traces now. Calling `/` was a source of flake, as the last trace was sometimes missing.